cirq/sim/clifford/clifford_simulator.py

```diff
@@ -279,6 +280,25 @@ def __init__(self, qubit_map, initial_state=0):
         self.ch_form = stabilizer_state_ch_form.StabilizerStateChForm(
             self.n, initial_state)
 
+    def _json_dict_(self):
+        return {
+            'cirq_type': self.__class__.__name__,
+            'qubit_map': dict([(v, k) for k, v in self.qubit_map.items()]),
```

Comment:
use a dictionary literal

```python
{v: k for k, v in ...}
```

Why are you reversing the keys and values? this might be dangerous? instead store as a list of tuples

Reply:
Done.